src/status_im/contexts/communities/actions/accounts_selection/style.cljs

```diff
@@ -20,9 +17,10 @@
   []
   {:position           :absolute
    :background-color   (colors/theme-colors colors/white colors/neutral-95)
+   :padding-bottom     (safe-area/get-bottom)
```

Comment:
Small suggestion: There are many instances in the app where we treat the value returned by `get-bottom` as static, also because we actually use `StaticSafeAreaInsets` behind the scenes. We could do the same here by moving the call out of this style function.